tests/cases/fourslash/documentHighlightsInvalidModifierLocations.ts

```diff
@@ -4,7 +4,12 @@
 ////    m([|readonly|] p) {}
 ////}
 ////function f([|readonly|] p) {}
+////
+////class D {
+////    m([|public|] p) {}
+////}
+////function g([|public|] p) {}
 
 for (const r of test.ranges()) {
-    verify.documentHighlightsOf(r, test.ranges());
+    verify.documentHighlightsOf(r, [r]);
```

Comment:
I changed the assertion here—the original assertion didn’t make a lot of sense and it was inconsistent with `public` and `private` modifiers. I expanded the test file content to show that.